modules/custom/az_event/az_event_trellis/az_event_trellis.install

```diff
@@ -11,3 +11,11 @@
 function az_event_trellis_update_1020801() {
   \Drupal::service('module_installer')->install(['views_remote_data']);
 }
+
+/**
+ * Ensure az_recurring_import_rule entity type is installed.
+ */
+function az_event_trellis_update_1020802() {
+  $entity_type_definition = \Drupal::service('entity_type.manager')->getDefinition('az_recurring_import_rule');
+  \Drupal::entityDefinitionUpdateManager()->installEntityType($entity_type_definition);
```

Comment:
Perhaps check if the entity type needs to be created?

```suggestion
  $changes = \Drupal::entityDefinitionUpdateManager()->getChangeList();
  $status = $changes['az_recurring_import_rule']['entity_type'] ?? '';
  // Determine if recurring import rule entity needs to be created.
  if ($status === EntityDefinitionUpdateManagerInterface::DEFINITION_CREATED) {
    $entity_type_definition = \Drupal::service('entity_type.manager')->getDefinition('az_recurring_import_rule');
    \Drupal::entityDefinitionUpdateManager()->installEntityType($entity_type_definition);
  }
```

(if included, `use Drupal\Core\Entity\EntityDefinitionUpdateManagerInterface;` would also need to be added so the constant is defined)